light-client/tests/light_client.rs

```diff
@@ -254,23 +254,12 @@ fn run_bisection_test(tc: TestBisection<LightBlock>) {
 }
 
 fn run_single_step_tests(dir: &str) {
-    // TODO: this test need further investigation:
-    let skipped = ["commit/one_third_vals_don't_sign.json"];
-
     let paths = fs::read_dir(PathBuf::from(TEST_FILES_PATH).join(dir)).unwrap();
 
     for file_path in paths {
         let dir_entry = file_path.unwrap();
         let fp_str = format!("{}", dir_entry.path().display());
 
-        if skipped
-            .iter()
-            .any(|failing_case| fp_str.ends_with(failing_case))
-        {
-            println!("Skipping JSON test: {}", fp_str);
-            return;
```

Comment:
ouch, this should have been `continue` instead of `return`

Reply:
I think I introduced this and it was copied over and while it was fixed in the `lite` module, I didn't noticed the same here.

https://github.com/informalsystems/tendermint-rs/blob/27298f0e7a5089096893f097d910f3e9271bf850/tendermint/tests/lite.rs#L251-L256